aiplatform/snippets/embeddings.go

```diff
@@ -28,40 +27,34 @@ import (
 	"google.golang.org/protobuf/types/known/structpb"
 )
 
-func embedTexts(
-	project, location string, texts []string) ([][]float32, error) {
+func embedTexts(project, location, model string) ([][]float32, error) {
```

Comment:
issue: since we're refactoring this sample, let's remove the return type.

See:
https://googlecloudplatform.github.io/samples-style-guide/#result

Reply:
This sample also needs to accept a `io.Writer` object as its first parameter.

Reply:
Regarding `io.Writer` - function doesn't print anything.

I think this is that rare case, when return is neat

> Methods should avoid a return type whenever possible

Cause otherwise there is a need to print two-dimensional array into buffer and later on somehow parse it back from the string. It seems much harder, with additional complexity.
Probably, I am missing something, do you any suggestion about that?